salt/config/__init__.py

```diff
@@ -2835,7 +2835,7 @@ def is_profile_configured(opts, provider, profile_name, vm_=None):
     alias, driver = provider.split(':')
 
     # Most drivers need an image to be specified, but some do not.
-    non_image_drivers = ['nova', 'virtualbox']
+    drivers_requiring_an_image = ['nova', 'virtualbox']
```

Comment:
These are drivers that don't require an image. Why rename this to ``drivers_requiring_an_image`` ?